### PR TITLE
[PCFG-56] Deploy and TLS cert api config fe

### PR DESCRIPTION
**Pull request by aomegax**
### List of changes

- removed tls cert for api-config
- added local vars for deploy pipeline

### Type of changes

- [ ] Add new pipeline
- [X] Update pipeline configuration
- [ ] Remove pipeline

### :warning: If it's new pipeline with code review have you added pagopa-github-bot -> Role: admin?

- [ ] Yes
- [X] No

### Other information

---

### CODE REVIEW

**Review by pasqualedevita — APPROVED** [review]

File: azure-devops/projects/pagopa-projects/tslcert-prod-config-prod-plaform-pagopa-it.tf [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      organization   = "pagopa"
+      name           = "le-azure-acme-tiny"
+      branch_name    = "master"
+      pipelines_path = "."
+    }
+    pipeline = {
+      enable_tls_cert         = true
+      path                    = "TLS-Certificates\\PROD"
+      dns_record_name         = ""
+      dns_zone_name           = "config.platform.pagopa.it"
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-projects/tslcert-prod-config-prod-plaform-pagopa-it.tf:13]:
```suggestion
      dns_record_name         = "config"
      dns_zone_name           = "platform.pagopa.it"
```
dns zone is `platform.pagopa.it` and the record `config`

File: azure-devops/projects/pagopa-projects/tslcert-uat-config-uat-plaform-pagopa-it.tf [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      organization   = "pagopa"
+      name           = "le-azure-acme-tiny"
+      branch_name    = "master"
+      pipelines_path = "."
+    }
+    pipeline = {
+      enable_tls_cert         = true
+      path                    = "TLS-Certificates\\UAT"
+      dns_record_name         = ""
+      dns_zone_name           = "config.uat.platform.pagopa.it"
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-projects/tslcert-uat-config-uat-plaform-pagopa-it.tf:13]:
```suggestion
      dns_record_name         = "config"
      dns_zone_name           = "uat.platform.pagopa.it"
```

File: azure-devops/projects/pagopa-projects/tslcert-dev-config-dev-plaform-pagopa-it.tf [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      organization   = "pagopa"
+      name           = "le-azure-acme-tiny"
+      branch_name    = "master"
+      pipelines_path = "."
+    }
+    pipeline = {
+      enable_tls_cert         = true
+      path                    = "TLS-Certificates\\DEV"
+      dns_record_name         = ""
+      dns_zone_name           = "config.dev.platform.pagopa.it"
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-projects/tslcert-dev-config-dev-plaform-pagopa-it.tf:13]:
```suggestion
      dns_record_name         = "config"
      dns_zone_name           = "dev.platform.pagopa.it"
```

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

**Comment by pasqualedevita** [comment]
Also a typo on file name `tslcert` instead of `tlscert`

**Review by pasqualespica — APPROVED** [review]
👍

**Comment by aomegax** [comment]
ping @pasqualedevita

**Review by pasqualedevita — APPROVED** [review]

**Review by pasqualedevita — APPROVED** [review]

**Pull request merged**